### ISSUE TITLE
Low contrast in "Manage" and user profile buttons in focus state

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#391]
<img width="482" alt="Screenshot 2023-03-04 at 08 36 18" src="https://user-images.githubusercontent.com/19519457/222885924-6cb11ba4-3514-449c-8fbd-dcbd6f5923ce.png">

In the focused state, the text on the "Manage" and user profile button doesn't meet colour contrast requirements. This is because our focus states are being partially applied here - the `background-color` is set, but the `color` is not.

Admin theme (Gin?) has:
```css
.toolbar .toolbar-bar .toolbar-item {
    color: #fff;
}
```

but 
```css
.toolbar .toolbar-bar {
    color: #ddd;
    background-color: #0f0f0f;
    box-shadow: -1px 0 3px 1px rgba(0,0,0,0.3333);
}
```

so on `a.toolbar-item` elements, the `a:focus` `background-color` takes precedence over Gin's defaults (as Gin's defaults are inherited from a parent), but `a:focus` `color` has no effect because there's a more specific CSS rule in place. This combination means we end up with white text on yellow.

We've seen similar issues crop up in three other places too at Essex, in our own theme - it's fairly common to set `background-color` on a parent element and then just `color` on the specific element, which results in the same problem. I don't know that there's anything much to be done in general, but it's just something to be aware of.

PR on the way to fix this specific issue.
